presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java

```diff
@@ -552,6 +556,15 @@ protected Void visitShowTables(ShowTables node, Integer context)
         }
 
         @Override
+        protected Void visitShowCreateTable(ShowCreateTable node, Integer context)
+        {
+            builder.append("SHOW CREATE TABLE ")
+                    .append(node.getName());
```

Comment:
Use `formatName()` here (lots of other places are broken, but we should add new ones correctly)